src/middleware/checkBodyParams.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+function checkBodyParams(paramsTitle, statusCode = 400) {
```

Comment:
Add an 'options' parameter to the function, as it has more than 3 parameters.

Reply:
I do not understand what is mean.. it like => function checkBodyParams(paramsTitle, statusCode = 400, options) {